charts/global-network-policies/templates/allow-to-dns.yaml

```diff
@@ -15,6 +15,14 @@ spec:
       networking.gardener.cloud/to-dns: allowed
   egress:
   - to:
+    {{- if .Values.dnsServer }}
+    - ipBlock:
+        cidr: {{ .Values.dnsServer }}/32 # required for node local dns feature, allows egress traffic to kube-dns
+    {{- end }}
+    {{- if .Values.nodeLocalIPVSAddress }}
+    - ipBlock:
+        cidr: {{ .Values.nodeLocalIPVSAddress }}/32 # required for node local dns feature, allows egress traffic to node local dns cache
+    {{- end }}
```

Comment:
Thanks! Those two fields are now always added even if the feature gate is disabled. Should this be taken into account?